config-cell-cleanup.html.md.erb

```diff
@@ -39,36 +39,14 @@ Selecting the best option for disk cleanup depends on the workload that the Dieg
 
 For PCF installations that primarily run buildpack-based apps, Pivotal recommends selecting the **Routinely clean up Cell Disk space** option. The **Routinely clean up Cell Disk space** option ensures that when a new stack becomes available on a cell, the old stack is dropped immediately from the cache.
 
-For PCF installations that primarily run Docker images, or both Docker images and buildpack-based apps, Pivotal recommends selecting the **Clean up Disk space once a threshold is reached** option with a reasonable threshold.
+For PCF installations that primarily run Docker images, or both Docker images and buildpack-based apps, Pivotal recommends selecting the **Clean up Disk space once usage fills disk** option with a reasonable reserve.
 
-### <a id='choosing-a-threshold'></a> Calculating a Threshold
+### <a id='choosing-a-reserve'></a> Calculating a Reserve
 
-To calculate a realistic value when configuring the disk cleanup threshold, you must identify some of the most frequently used Docker images in your PCF installation. Docker images tend to be constructed by creating layers over existing base images. In some cases, you may find it easier to identify which base Docker images are most frequently used.  
-
```

Comment:
are there any more specific numbers we can give this -- how much of an increase do we recommend? or could we provide an example?